config/variables/adiltouati-main.yml

```diff
@@ -0,0 +1,285 @@
+# ----------------------------------------------------------------------------------
```

Comment:
Can you remove your environment config?

Reply:
Sure, I believe I did it in the new update of the PR, but will do